crowbar_framework/app/views/barclamp/glance/_edit_attributes.html.haml

```diff
@@ -16,6 +16,8 @@
       %legend
         = t(".store.file_header")
       = string_field :filesystem_store_datadir
+      .alert.alert-info{ "data-show-for-clusters-only" => "true", "data-elements-path" => "glance-server" }
+        = t('.store.file_ha_info')
```

Comment:
> Actually, I would put the message below the "Default Storage Store" select, and only if the value there is "File".

This would require changing the module taking care of hiding/showing cluster-related info. Is it worth it?